src/remote-config/remote-config-api.ts

```diff
@@ -354,7 +354,21 @@ export interface ServerTemplateData {
 /**
  * Represents optional arguments that can be used when instantiating {@link ServerTemplate}.
  */
-export interface ServerTemplateOptions {
+export interface GetServerTemplateOptions {
```

Comment:
Does it makes sense to create a base type and extend from it?

```
export interface InitServerTemplateOptions extends GetServerTemplateOptions {
      template?: ServerTemplateData,
}
```

Reply:
Makes sense. I'll update it.